src/components/NetworkPill.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Icon } from '@stellar/design-system';
+import { useWallet } from '../hooks/useWallet';
+import { stellarNetwork } from '../contracts/util';
+
+const NetworkPill: React.FC = () => {
+  const { network, address } = useWallet();
+  
+  // Determine network display name
+  const networkName = stellarNetwork || 'local';
```

Comment:
You can use `stellarNetwork` directly, it has fallbacks in its declaration in that other file